peachjam/js/components/FindDocuments/index.vue

```diff
@@ -387,9 +490,19 @@ export default {
           this.facets.forEach((facet) => {
             params.append('facet', facet.name);
           });
-
+          Object.keys(this.advancedFields).forEach(key => {
+            const value = this.advancedFields[key];
+            if (!value) return;
+            if (key === 'date' && value.date_from && value.date_to) {
+              const dateFrom = moment(value.date_from).locale(getUserLocale()).format('YYYY-MM-DD');
```

Comment:
what does adding `.locale()` do?

Reply:
my reasoning for this was that different locales present different date formats. Setting `moment.locale()` will allow `moment` to know how to parse the date according to the associated locale. 

However we actually dont need to do this because moment uses the locale of the browser by default..
so i've removed this syntax